### PR TITLE
Air vendor ui changes and fixes

### PR DESCRIPTION
**Pull request by Sovexe**
## About the PR 
* Modifies the air vendor window, replaces the number input with a slider
* Slider fill level is set to tank's current pressure to ease in gauging how much air is being added
* `vend_type` is displayed on the UI to support the new plasma vendor
* Vending free air is fixed for podwars
* Max and Min buttons now disable when pressure is set to each respectively
* minor cleanup of var ordering and adding type check to `handleChangePressure`

image

## Why's this needed? 
* bugfixes
* improve user experience

### CODE REVIEW

File: tgui/packages/tgui/interfaces/AirVendor.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
  * @copyright 2023
- * @author Garash (https://github.com/Garash2k)
+ * @author Original Garash (https://github.com/Garash2k)
+ * @author Changes Sovexe (https://github.com/Sovexe)
  * @license ISC
  */
 
 import { useBackend } from "../backend";
-import { LabeledList, NumberInput, Button, Section, Dimmer } from "../components";
+import { Button, Dimmer, Flex, LabeledList, Section, Slider } from "../components";
```
Comment (Garash2k) [inline tgui/packages/tgui/interfaces/AirVendor.tsx:10]: Ooh, alphabetical! I wonder if there's a lint rule that could validate and order them automatically on save.

File: tgui/packages/tgui/interfaces/AirVendor.tsx [hunk 2]
```diff
@@ -15,48 +16,73 @@ type AirVendorParams = {
   cash: number,
   cardname: string,
   bankMoney: number,
-
+  vend_type:string,
```
Comment (Garash2k) [inline tgui/packages/tgui/interfaces/AirVendor.tsx:19]: Super minor point: inconsistent space.

**Review by Garash2k — APPROVED** [review]

File: tgui/packages/tgui/interfaces/AirVendor.tsx [hunk 3]
```diff
@@ -15,48 +16,73 @@ type AirVendorParams = {
...
-            animated
-            width={6}
-            value={target_pressure}
-            minValue={min_pressure}
-            maxValue={max_pressure}
-            onChange={(_e, new_pressure) => handleChangePressure(new_pressure)} />
-          <Button
-            onClick={() => handleChangePressure(max_pressure)}
-            content="Max" />
+          <Flex>
```
Comment (Garash2k) [inline tgui/packages/tgui/interfaces/AirVendor.tsx:61]: Would `<Stack>` be appropriate here?
https://github.com/goonstation/goonstation/blob/master/tgui/docs/component-reference.md#stack

Comment (Sovexe) [reply tgui/packages/tgui/interfaces/AirVendor.tsx]: implemented

**Review by Garash2k — APPROVED** [review]

**Review by mordent-goonstation — APPROVED** [review]

**Pull request merged**